ssn/rdf/examples/InkBird.ttl

```diff
@@ -0,0 +1,50 @@
+
+
+<https://example.org/sensor/IBS-TH2> rdfs:subClassOf sosa:Sensor ;
+    rdfs:comment "The class of IBS-TH2 sensors is a subclass of the general class of sensors" ;
+    gs1:pip <https://inkbird.com/products/hygrometer-ibs-th2> ;
+    rdfs:label "Bluetooth Temperature and Humidity Sensor IBS-TH2" ;
+    skos:notation "IBS-TH2" ;
+    sosa:observes <http://qudt.org/vocab/quantitykind/RelativeHumidity> ;
```

Comment:
The range of sosa:observes is sosa:Sensor, not owl:Class.

Reply:
The range of `sosa:observes` is `sosa:Property` - https://w3c.github.io/sdw-sosa-ssn/ssn/#SOSAobserves

Reply:
Alright. But [quantitykind:RelativeHumidity](http://qudt.org/vocab/quantitykind/RelativeHumidity) is neither.

Reply:
Conceptually, `qudt:QuantityKind` is a subclass of `sosa:Property`.
I did not state it explicitly here, but in ssn-observation.ttl there is a guarded constraint on the class `sosa:Sensor` that `sosa:observes` takes values from `sosa:Property`. 
Thus, a reasoner can infer that `quantitykind:RelativeHumidity` is a member of `sosa:Property`. 

But I guess it wouldn't do any harm to say this explicitly. 
Done.